### PR TITLE
Upgrade psammead-styles to emotion 11

### LINKED ISSUE #8348: Upgrade @emotion/core and @emotion/styled to new major version 11 in Simorgh
**Is your feature request related to a problem? Please describe.**
Upgrade `@emotion/core` and `@emotion/styled` to new major version 11.

Psammead PR 1: https://github.com/bbc/psammead/pull/4193
Psammead PR 2 (psammead-styles): https://github.com/bbc/psammead/pull/4231

Details of any breaking changes can be found here https://github.com/emotion-js/emotion/releases/tag/%40emotion%2Fstyled%4011.0.0

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves https://github.com/bbc/simorgh/issues/8348

**Overall change:** 

- Upgrade psammead-styles to emotion 11 & remove remaining emotion 10 dependencies from base Psammead

**Code changes:**

- Upgrade psammead-styles to emotion 11 (from `"@emotion/core": "^10.0.35"` to `"@emotion/react": "^11.1.4"`)
- Remove `"@emotion/core": "^10.1.1"` from `./package.json` 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.69 | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Upgrade psammead styles to Emotion v11 & remove Emotion 10 packages from root dependencies |
```
Comment (RichardPK) [inline CHANGELOG.md]:
```suggestion
| 4.0.69 | [PR#4231](https://github.com/bbc/psammead/pull/4231) Upgrade psammead styles to Emotion v11 & remove Emotion 10 packages from root dependencies |
```

File: packages/utilities/psammead-styles/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 7.0.4 | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Upgrade psammead styles to Emotion v11 & remove Emotion 10 packages from root dependencies |
```
Comment (RichardPK) [inline packages/utilities/psammead-styles/CHANGELOG.md]:
```suggestion
| 7.0.4 | [PR#4231](https://github.com/bbc/psammead/pull/4231) Upgrade psammead styles to Emotion v11 & remove Emotion 10 packages from root dependencies |
```

**Review by jroebu14 — APPROVED** [review]

File: packages/utilities/psammead-styles/package.json [hunk 3]
```diff
@@ -25,6 +25,6 @@
   ],
   "peerDependencies": {
     "react": "^16.13.1",
-    "@emotion/core": "^10.0.35"
+    "@emotion/react": "^11.1.4"
```
Comment (pharingee) [inline packages/utilities/psammead-styles/package.json:28]: Why was this changed? The `psammead-styles` package uses `@emotion/core` and `@emotion/styled`

**Review by pharingee — APPROVED** [review]

File: packages/utilities/psammead-styles/package.json (hunk 3 above)
Comment (RichardPK) [reply packages/utilities/psammead-styles/package.json:28]: Hang on good point. Did I not change the imports within the component? Going to quickly put a DNM label on this

**Comment by RichardPK** [comment]
DNM - gotta change an import.

**Review by pharingee — CHANGES_REQUESTED** [review]
Unapproving to fix @emotion issue

File: packages/utilities/psammead-styles/package.json (hunk 3 above)
Comment (RichardPK) [reply packages/utilities/psammead-styles/package.json:28]: Removed the import of @emotion/core for psammead-styles and replaced it with @emotion/react

File: packages/utilities/psammead-styles/src/__snapshots__/global-styles.test.jsx.snap [hunk 4]
```diff
@@ -4,213 +4,248 @@ exports[`global-styles should render global styles 1`] = `
 <head>
   <style
     data-emotion="css-global"
+    data-s=""
```
Comment (RichardPK) [inline packages/utilities/psammead-styles/src/__snapshots__/global-styles.test.jsx.snap:7]: 72 line changes in this snapshot are additions of this blank `data-s` property .

The `sourceMappingUrl` is also being removed for every entry - which I think is a good thing. This was added as a negative side effect of the initial emotion migration

**Review by jroebu14 — APPROVED** [review]

**Review by pharingee — APPROVED** [review]
